tools/clang/test/CodeGenSPIRV/spirv.debug.opline.intrinsic.vulkan1.1.hlsl

```diff
@@ -0,0 +1,62 @@
+// Run: %dxc -T cs_6_0 -E main -Zi -fspv-target-env=vulkan1.1
```

Comment:
This test was not added to `CodeGenSpirvTest.cpp`

Reply:
added.